src/containers/home/home.component.jsx

```diff
@@ -1,49 +1,44 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 
 import { auth } from "database";
 
 import { fetchPosts } from "database/posts";
-import { addPost } from "database/addPost";
 
 import { useSelector } from "react-redux";
+import { POINT_CONVERSION_COMPRESSED } from "constants";
```

Comment:
What is this constant for?

Reply:
resolved in second PR, because of auto import, if I hit enter on an autocomplete, this happens and I dont notice sometimes